using_images/s2i_images/dot_net_core.adoc

```diff
@@ -33,17 +33,17 @@ and Python from within {product-title}.
 [[dot-net-core-supported-versions]]
 == Supported Versions
 
+* .NET Core version 2.2
 * .NET Core version 2.1
-* .NET Core version 2.0
 * .NET Core version 1.1
 * .NET Core version 1.0
 * Supported on Red Hat Enterprise Linux (RHEL) 7
 ifdef::openshift-enterprise[]
 and {product-title} versions 3.3 and later
 endif::openshift-enterprise[]
 
-For release details related to .NET Core version 2.1, see
-link:https://access.redhat.com/documentation/en-us/net_core/2.1/html/release_notes_for_containers/[Release
+For release details related to .NET Core version 2.2, see
```

Comment:
We currently have release notes for 2.2 and 1.1 and 1.0. Only version 2.1 doesn't have a link to release notes any more. Is that intentional? Should we keep a link to the 2.1 release notes?

Reply:
Yes, I may have been confusing in my notes for changes to make. Until 2.1 is EOL'd it is probably best to keep the 2.1 release notes available.

Reply:
Also, Ashley, we will have one or two new environment variables to add to the table. I will supply this new information to you.